.vscode/SETUP.md

```diff
@@ -0,0 +1,20 @@
+## Setup
```

Comment:
Is there any harm in placing the `vscode` details in the main `README`?

Reply:
Most of the `README` is dedicated to getting the stack up and running, with little emphasis on development environment.  Maybe we need a dedicated development environment section?